Adding apps to pbcs (#1203)
spryker/spryker-docs: docs/pbc/all/payment-service-providers/payone/integrate-payone.md

---
title: Integrate Payone
description: Learn how you can integrate the Payone app into your Spryker shop
template: howto-guide-template
---

To integrate Payone, follow these steps:

1. In your store's Back Office, go to **Apps&nbsp;<span aria-label="and then">></span> Catalog**.
2. Click **Payone**.

>    This takes you to the Payone app details page.

Reviewer: Wrap page names in bold

Author: this is just how we call the page, analogously to "product details page".

3. In the top right corner of the Payone app details page, click **Connect app**.
   This displays a message about the successful connection of the app to your SCCOS. The Payone app's status changes to *Connection pending*.   
4. Go to [Payone](https://www.payone.com?ref=spryker-documentation) and obtain the credentials.
   
   {% info_block infoBox "Info" %}

   It takes some time to obtain credentials from Payone because you have to go through a thorough vetting process by Payone, such as the "know your customer" (KYC) process before Payone verifies you.

   {% endinfo_block %}

> 5. Go back to your store's Back Office, to the Payone app details page.

Reviewer: Since there is a time gap due to the verification, consider splitting the instructions into two parts, the second one starting from this step. You can call them `Connect Payone` and `Configure Payone`.

6. In the top right corner of the Payone app details page, click **Configure**.
7. On the Payone app details page, fill in fields in the **Credentials** section.
   ![payone-app-detais](https://spryker.s3.eu-central-1.amazonaws.com/docs/aop/user/apps/payone/payone-app-details.png)

> 8. Select *Payone Environment Mode*: **TEST** or **LIVE**.

Reviewer: `Payone Environment Mode` as a UI element should be in bold. Also, I don't see a screenshot, so make sure to reflect the UI's case.

Reviewer: Usually, you don't need to list the options. If you do, there's a reason to believe readers will not understand what the options mean. Then, you need to explain all of them. However, in this case, the options sound quite self-explanatory.

> 9. Enter your *Shop Name*. This name will be displayed on **Payment Page** as a merchant label for whom to pay:

Reviewer: Also, I'm not sure what `**Payment Page**` refers to. Might it be `**Payment** page` instead?

   ![payone-shop-name](https://spryker.s3.eu-central-1.amazonaws.com/docs/aop/user/apps/payone/payone-shop-name.png)

> 10. Select a payment method⁠—either **CREDIT CARD** or **PAYPAL**, or both.

Reviewer: Same here regarding options. If you want to specify that the reader can select both options, you can say `Select one or more payment methods`.

   ![payone-payment-methods](https://spryker.s3.eu-central-1.amazonaws.com/docs/aop/user/apps/payone/payone-payment-methods.png)
11. Optional: To configure payment methods per store, click **Payment methods per store configuration** and select stores for the defined payment methods.
12. Click **Save**.
   

> In case the app was connected successfully, a corresponding message appears, and the app status changes to **Connected**. The payment methods, which you selected in step 8, appear in **Administration&nbsp;<span aria-label="and then">></span>  Payment methods**:

Reviewer: ```suggestion
If the app was connected successfully, a corresponding message appears, and the app's **STATUS** changes to **Connected**. The payment methods you've selected in step 8, appear in **Administration&nbsp;<span aria-label="and then">></span>  Payment methods**:
```
